packages/core/src/modules/presentation-exchange/PresentationExchangeModule.ts

```diff
@@ -0,0 +1,16 @@
+import type { DependencyManager, Module } from '../../plugins'
+
+import { PresentationExchangeService } from './PresentationExchangeService'
+
+/**
+ * @public
+ */
+export class PresentationExchangeModule implements Module {
```

Comment:
maybe we can add the experimental warning to this module as well? As we'll need to make several changes to this over the coming weeks